contrib/ocm/machine-pools/main.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    ocm = {
+      version = ">= 1.0.1"
+      source  = "terraform.local/local/ocm"
```

Comment:
The provider name and registry path changed to `rhcs`
Please change in all relevant places

Reply:
Fix